### PR TITLE
Add support for VkRenderPassInputAttachmentAspectCreateInfo

### PR DESCRIPTION
**Pull request by hakzsam**

### CODE REVIEW

File: fossilize.cpp [hunk 1]
```diff
@@ -889,6 +892,18 @@ static void hash_pnext_struct(const StateRecorder *,
 	h.u32(info.reductionMode);
 }
 
+static void hash_pnext_struct(const StateRecorder *,
+                              Hasher &h,
+                              const VkRenderPassInputAttachmentAspectCreateInfo &info)
+{
+	h.u32(info.aspectReferenceCount);
+	for (uint32_t i = 0; i < info.aspectReferenceCount; i++) {
```
Comment (HansKristian-Work) [inline fossilize.cpp:900]: Brace on next line.

File: fossilize.cpp [hunk 2]
```diff
@@ -3702,6 +3721,30 @@ bool StateReplayer::Impl::parse_sampler_reduction_mode(const Value &state,
 	return true;
 }
 
+bool StateReplayer::Impl::parse_input_attachment_aspect(const Value &state,
+						        VkRenderPassInputAttachmentAspectCreateInfo **out_info)
+{
+	auto *info = allocator.allocate_cleared<VkRenderPassInputAttachmentAspectCreateInfo>();
+	*out_info = info;
+
+	info->aspectReferenceCount = state["aspectReferenceCount"].GetUint();
```
Comment (HansKristian-Work) [inline fossilize.cpp:3730]: This is implied by state["aspectReferences"].Size().

Comment (hakzsam) [reply fossilize.cpp:3730]: Should I remove aspectReferenceCount and rely on the array size?

**Review by HansKristian-Work — CHANGES_REQUESTED** [review]
Minor nits.

File: fossilize.cpp (hunk 2 above)
Comment (HansKristian-Work) [reply fossilize.cpp:3730]: Yes.

Comment (hakzsam) [reply fossilize.cpp:3730]: Well, it must be greater than 0 according to the spec.

Comment (HansKristian-Work) [reply fossilize.cpp:3730]: Ok, probably ignore the test about 0 count then.

File: fossilize.cpp (hunk 1 above)
Comment (hakzsam) [reply fossilize.cpp]: Fixed.

File: fossilize.cpp (hunk 2 above)
Comment (hakzsam) [reply fossilize.cpp]: Should be addressed now hopefully.

**Review by HansKristian-Work — APPROVED** [review]
LGTM

**Pull request merged**